src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -196,6 +194,17 @@ module Types = struct
                 ("Histograms", Histograms.to_text histograms) :: acc )
           ~consensus_mechanism:(fun acc x ->
             ("Consensus Mechanism", f x) :: acc )
+          ~consensus_configuration:(fun acc x ->
+            let render conf =
+              match Consensus.Configuration.to_yojson conf with
+              | `Assoc ls ->
+                  List.fold_left ls ~init:"" ~f:(fun acc (k, v) ->
```

Comment:
It doesn't really matter here because the strings are so small but this is an `n^2` algorithm due to string copies on every concat. I don't see anything convenient lying around equivalent to a java `StringBuilder` though.

Reply:
Yeah, I'm aware. I wish every language compiler handled strings the way erlang does. Not super familiar with `StringBuilder` in Java, but I think it's similar.